internal/pkg/miniprocfs/testdata/1920080/exe

```diff
@@ -1 +1 @@
-/usr/bin/fish
```

Comment:
this fixes some weirdness with `go-git` based checkouts of our repo, it doesn't like absolute symlinks

this is all testdata, and tests don't care, so why not fix it